### PR TITLE
ENH: Add Addis Ababa University, Ethiopia

### PR DESCRIPTION
**Pull request by dragonyanglong**
redo edits in PR #77 , and submit PR again. Close Issue #61.

### CODE REVIEW

File: db/institutions.yml [hunk 1]
```diff
@@ -486,3 +486,12 @@ ubelgrade:
   name: University of Belgrade
   street: Studentski trg 1
   zip: 11000
+aau:
+  aka: [Addis Ababa University, አዲስ አበባ ዩኒቨርሲቲ, AAU]
```
[the remark this answers is not in the document]
Comment (dragonyanglong) [reply db/institutions.yml]: Thanks. I deleted the name and added its old names into aka that were used in the history.

**Comment by dragonyanglong** [comment]
Just rebased, it is ready for merge now. Please review @sbillinge

**Comment by sbillinge** [comment]
Thanks @dragonyanglong !   btw, this didn't close the issue.  You have to say "closes #N" for it to automatically close (there are a few other syntaxes that work too, but only specific ones.

**Comment by dragonyanglong** [comment]
Closes #61

**Pull request merged**